### DISCUSSION TITLE
Express Authentication

### DISCUSSION THREAD

**Discussion by kriptonian1** [kriptonian1/College-Partner#2] [Q&A] [answered]
How can I implement express authentication

**Comment by bartick** [comment, marked as answer]
To implement a express authentication you need to make sure all your request pass through a middlewire that will determine if the request needs to passed on more or need to be disregarded. This is what an authentication is.

For Example:
```js
module.exports = (req, _res, next) => {
    if (req.method==='GET') {
        next();
        return;
    }
   else {
         // Your suitable condition to pass though.
   }
```
